storage/objects/stream_file_upload.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 Google LLC
```

Comment:
Should this be `Copyright 2021 Google LLC`?

Reply:
good catch!